app/abr-menu.js

```diff
@@ -30,6 +31,10 @@ function preprocessTemplate (element, config) {
             win.webContents.send("command", command, parameters);
         },
         replaceAttributes = function (item) {
+            if (item.label) {
+                item.label = localize.get(config, item.label);
```

Comment:
Instead of using the `label` key (which is a bit tricky in my opinion), you can add an `id` to menu-items (it is actually [supported by Electron](https://github.com/electron/electron/blob/master/docs/api/menu-item.md)) and keep the label as a default value for missing translations (which fixes one of the TODOs your wrote about).

For instance:

```javascript
if (item.id) {
	item.label = localize.get(lang, item.id);
}
```

Another benefit: you don't need to prefix your keys with `%` anymore.